testData/goto/TypeDef2.hx

```diff
@@ -1,7 +1,7 @@
-typedef ExtendedString<T> = {>String, key:T};
+typedef ExtendedString<T> = {key:T};
```

Comment:
Why did you change this test?  It used to test typedef extensions (e.g. all fields of String were available in ExtendedString).

Hmm.  I guess it didn't because it was using 'key' instead of some value from String.

I just want to be sure that we still have tests that use the "inherited field" syntax.

Reply:
I've changed it because String is not anonymous type, and it can't be used as parent type.
I'll check tests.

Reply:
I've reverted test, but it's not valid for real, anonymous type can't extend extern classes.